sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample3_ExtractKeyPhrases.md

```diff
@@ -19,7 +19,7 @@ To extract key phrases from a document, use the `ExtractKeyPhrases` method.  The
 ```C# Snippet:ExtractKeyPhrases
 string document = "My cat might need to see a veterinarian.";
 
-IReadOnlyCollection<string> keyPhrases = client.ExtractKeyPhrases(document).Value;
+KeyPhraseCollection keyPhrases = client.ExtractKeyPhrases(document);
```

Comment:
Per recent guidance:  **No implicit casts in our samples**

Reply:
wait, what? what did I miss and where is it? I can't find it in the guidelines.

Reply:
Found information. Will address later according to the decision